src/qibolab/instruments/qm.py

```diff
@@ -1015,3 +1015,6 @@ def sweep(self, qubits, sequence, options, *sweepers):
 
         result = self.execute_program(experiment)
         return self.fetch_results(result, qmsequence.ro_pulses)
+
+    def play_sequences(self, qubits, sequence, options):
+        pass
```

Comment:
```suggestion
        raise NotImplementedError
```
would also work, but not important since we will implement it soon.